docs/conf/portland/2019/news/welcome-get-ready.rst

```diff
@@ -0,0 +1,82 @@
+
+Get involved
+------------
+
+* **Plan a project for Writing Day.** The `Writing Day <https://www.writethedocs.org/conf/portland/2019/writing-day/>`_ will be held on Sunday from *9am-5pm*, and you can drop in at any time during the day. If you have a project you want to propose to others, you can pitch your topic during the welcome announcement that will be held at *9:30am*. Check out the `Writing Day <https://www.writethedocs.org/conf/portland/2019/writing-day/>`_ page for a few suggested projects!
+* **Plan a lightning talk.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want. This is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Get ready for the Job Fair**. Once again on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/2019/job-fair>`_ where you can find your next documentarian friendly job. The job fair will be held in Lola's Room from *9:40am* to *11:40am*.
+* **Plan an unconference session.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/2019/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a very specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
+* **Coffee, breakfast, and lunch are provided on all three days!** Don't worry about coffee before you come to the conference on Sunday, Monday, or Tuesday. Coffee and breakfast are served at 8:15, and the conference starts at 9.
+
+Find your way around
+--------------------
+
+* **Riding the trains.** Public transit is generally excellent in Portland, you can get from the airport downtown on the MAX red line. For more information, see our `visiting page <https://www.writethedocs.org/conf/portland/2019/visiting/>`_ for all the details you need to get around.
+* **Riding a bike.** Portland has a `bike share program <https://www.biketownpdx.com/>`_ that is having a promotion in May that includes free riding! Check it out while you're in town.
+
+Have fun
+--------
+
+* **Join us for a walk in the woods**. If you're planning to come out for the `hike <https://www.writethedocs.org/conf/portland/2019/hike/>`_ on Saturday at *1:45pm*, make sure you `reserve your spot <https://ti.to/writethedocs/write-the-docs-portland-2019/>`_ . We're hoping for less hail than some other hikes we've done in the past, but bringing a rain jacket would be wise.
```

Comment:
```suggestion
* **Join us for a walk in the woods**. If you're planning to come out for the `hike <https://www.writethedocs.org/conf/portland/2019/hike/>`_ on Saturday at *1:45pm*, make sure you `reserve your spot <https://ti.to/writethedocs/write-the-docs-portland-2019/>`_ . Note that we've changed the route a bit this year because of construction, and will be doing an "out and back" to Pittock Mansion. 
```